core-primitives/test/src/mock/shielding_crypto_mock.rs

```diff
@@ -15,7 +15,7 @@
 
 */
 
-use itp_sgx_crypto::ShieldingCrypto;
+use itp_sgx_crypto::{ShieldingCryptoDecrypt, ShieldingCryptoEncrypt};
```

Comment:
So this mock can now also be made available in `std`, right? Maybe you can make that change as well here, should be a small thing. Just remove the `#[cfg(feature = sgx)]` guard

Reply:
Yes, I can do that. Should I do the same for the other key pairs as well, or rather in a different PR?

Reply:
The other key pairs should already be available in `std`, the shielding key was the last one only restricted to `sgx`